### PR TITLE
Add Tabs extended page

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues/893

### Changelog

**New**

- Tabs extended page

**Changed**

- removed react URL for Content group banner.

### CODE REVIEW

File: src/pages/components/content-group-banner.mdx [hunk 1]
```diff
@@ -28,6 +28,7 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 Since this component is intended for supplementary content, it is not intended to be used above the [Lead space](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/leadspace). Content group - banner accommodates a headline and one to two additional CTAs.
 
+
```
Comment (shixiedesign) [inline src/pages/components/content-group-banner.mdx:31]:
```suggestion
```

**Comment by shixiedesign** [comment]
FYI @JennySanchez @vpicone the max height of left nav is set at 93.75rem. Just need that bumped up. I know this might affect the animation though. So please check to adjust speed of the accordion if it applies. 

image

**Comment by JennySanchez** [comment]
> FYI @JennySanchez @vpicone the max height of left nav is set at 93.75rem. Just need that bumped up. I know this might affect the animation though. So please check to adjust speed of the accordion if it applies.
> 
> image

@shixiedesign The max height of the left nav was removed, and we should not have it as a blocker from now on.

**Comment by shixiedesign** [comment]
@JennySanchez ready for another review!

**Comment by JennySanchez** [comment]
@shixiedesign The page looks good. The link to React is taking me to the Tab extended page. If it's not available, it should have the Under construction label and Coming soon (see screenshot below).

Screen Shot 2021-05-14 at 3 20 33 PM

File: src/data/components.json [hunk 2]
```diff
@@ -917,7 +943,7 @@
       "description": "Content group — banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.",
       "storybook": {
         "webcomponents": "http://ibmdotcom-web-components.mybluemix.net/?path=/story/components-content-group-banner--default",
-        "react": "http://ibmdotcom-react.mybluemix.net/?path=/story/components-content-group-banner--default"
+        "react": " "
```
Comment (kennylam) [inline src/data/components.json:946]: @shixiedesign Removing the space will automatically insert `Coming soon`. Currently with the space, an empty link will be inserted instead.

image


```suggestion
        "react": ""
```

**Comment by JennySanchez** [comment]
I checked again and still can not see the `Under construction` label shows up via preview.
@shixiedesign @kennylam Can you see it?

Screen Shot 2021-05-17 at 4 52 17 PM

**Review by JennySanchez — APPROVED** [review]
We are good to go.

**Pull request merged**